Assets/MixedRealityToolkit/Graph/link.xml

```diff
@@ -0,0 +1,13 @@
+<linker>
+	<assembly fullname="System.Runtime.Serialization" preserve="nothing">
+		<type fullname="System.Runtime.Serialization.Configuration.DataContractSerializerSection" preserve="all" />
+		<type fullname="System.Runtime.Serialization.Configuration.SerializationSectionGroup" preserve="all" />
+		<type fullname="System.Runtime.Serialization.Configuration.DeclaredTypeElementCollection" preserve="all" />
+		<type fullname="System.Runtime.Serialization.XmlObjectSerializerWriteContext" preserve="all" />
+		<type fullname="System.Runtime.Serialization.XmlWriterDelegator" preserve="all" />
+	</assembly>
+
+	<assembly fullname="Microsoft.Identity.Client">
+		<type fullname="Microsoft.Identity.Client.Internal.*" preserve="all" />

```

Comment:
Can you explain what this does please?

Reply:
Nvm it was in the readme that it's to prevent stripping byte code.

Reply:
I will add an XML comment in place!